src/func_cache.c

```diff
@@ -274,6 +291,62 @@ static FuncInfo funcinfo[] = {
 		.group_estimate = time_bucket_group_estimate,
 		.sort_transform = time_bucket_sort_transform,
 	},
+
+	{
+		.is_timescaledb_func = true,
+		.is_bucketing_func = true,
+		.funcname = "time_bucket_gapfill",
+		.nargs = 4,
+		.arg_types = { INTERVALOID, TIMESTAMPOID, TIMESTAMPOID, TIMESTAMPOID },
+		.group_estimate = time_bucket_group_estimate,
+		.sort_transform = time_bucket_gapfill_sort_transform, // TODO change transform fn
```

Comment:
How is this todo important? Can it be fixed now or an issue created, which describes what should be done?

Reply:
and this one is obsolete